src/About/About.tsx

```diff
@@ -22,14 +22,48 @@ const About: React.FC = () => {
   return (
    <section className="about-section">
     <span className="intro-text">
-     <h2>Plan. Research. Experience.</h2>
+     <h2>Plan. Go. Breathe.</h2>
      <p>
-      At <b>Weather or Not</b>, we know how important it is to you and your
-      loved ones to have accurate weather and air quality information. <br />{" "}
+      At <b>Breathe AmAIRica</b>, we know how important it is for you and your
+      loved ones to have accurate weather and air quality information.
       <br />
-      Our data is updated every 10-60 minutes, giving you the most updated
-      conditions to help you plan for your enjoyment and peace of mind.
+      <br/>
+      Whether you're planning a trip or relocating across the country, or just a walk around the block with your dog, <br/>
```

Comment:
What do you think of moving the fact that we update every 10-60 minutes to the start of the sentence? Some users like to skim and I think it's awesome to recognize that fact first! What about like this:
"Our data is updated every 10-60 minutes, so whether you are planning a trip, relocating across the country or just taking a walk around the block with your dog, we give you the most updated conditions to help you make your next move!"